### DISCUSSION TITLE
[Question] storing internet from WiFi using shadowsocks.

### DISCUSSION THREAD

**Discussion by ShadowSocks12** [shadowsocks/shadowsocks-android#2593] [Q&A]
Hi, I have seen lots of videos talking about that shadowsocks stores internet from wifi, A lot of people say it is not working some say it is working, and some say it is a lie and some say other things about it......
so I came here to get an official answer from the developers so this headache ends please. I mean reading people's comments about storing internet.
Does shadowsocks android really store internet from WiFi and when disabling WiFi on phone and activating shadowsocks again, the stored internet will be used?
And another question please ........ in Shadowsocks android, what is the meaning of this arrow  ↑ and the meaning of this ↓ 
And Thank You Very Much.

**Comment by zkebami** [comment]
the official answer is no yu cant !

**Comment by dev4u** [comment]
#1390
